console_unix.go

```diff
@@ -16,19 +16,21 @@ func NewPty() (Console, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	if err := saneTerminal(f); err != nil {
-		return nil, "", err
-	}
 	slave, err := ptsname(f)
 	if err != nil {
 		return nil, "", err
 	}
 	if err := unlockpt(f); err != nil {
 		return nil, "", err
 	}
-	return &master{
+	m := &master{
 		f: f,
-	}, slave, nil
+	}
+	// getCurrent will set the current state on the pty master
+	if _, err := m.getCurrent(); err != nil {
```

Comment:
add it to `newMaster()` to so `ConsoleFromFile()`  get the original state at the time where it was called?

Reply:
done

Reply:
This could call `newMaster()` now :stuck_out_tongue: 

Also CI failure is weird, it seems that it run on a different commit than this one